draft-ietf-rtcweb-jsep.xml

```diff
@@ -3901,63 +3901,59 @@ v=0
 o=- 4962303333179871722 1 IN IP4 0.0.0.0
 s=-
 t=0 0
-a=group:BUNDLE a1 v1
 a=ice-options:trickle
+a=group:BUNDLE a1 v1
 m=audio 56500 UDP/TLS/RTP/SAVPF 96 0 8 97 98
 c=IN IP4 192.0.2.1
 a=mid:a1
-a=rtcp:56501 IN IP4 192.0.2.1
 a=msid:47017fee-b6c1-4162-929c-a25110252400
```

Comment:
Reading section 3.2.1 of https://tools.ietf.org/html/draft-ietf-mmusic-msid-15, it seems like the msid should have the "id" attribute of the MS set as app data on the msid line.